yt_idv/shaders/grid_expand.geom.glsl

```diff
@@ -36,22 +40,35 @@ uniform vec3 arrangement[8] = vec3[](
 
 uniform int aindex[14] = int[](6, 7, 4, 5, 1, 7, 3, 6, 2, 4, 0, 1, 2, 3);
 
+
 void main() {
 
     vec4 center = gl_in[0].gl_Position;
-
-    vec3 width = vright_edge[0] - vleft_edge[0];
-
+    vec3 width, le;
     vec4 newPos;
 
+    if (is_spherical){
```

Comment:
one more reminder for myself: go look at the pre-processor business again (e.g., https://github.com/yt-project/yt_idv/pull/89), might not need uniforms here.